adhocracy4/maps/fields.py

```diff
@@ -0,0 +1,41 @@
+from django.core import validators as django_validators
+from django.utils.translation import ugettext_lazy as _
+from jsonfield.fields import JSONField, JSONFormField
+
+
+class GeoJSONFormField(JSONFormField):
+
+    def to_python(self, value):
+        empty_featureset = '{"type":"FeatureCollection","features":[]}'
+        if (value not in django_validators.EMPTY_VALUES and not
+                value == empty_featureset):
+            return super().to_python(value)
+        else:
+            return None
+
+
+class GeoJSONField(JSONField):
```

Comment:
`JSONField` sets `help_text` to "Enter valid JSON" by default. This is not helpful in our case. I am not sure how  we can avoid this tough.